plugins/catalog-backend/src/ingestion/processors/GithubDiscoveryProcessor.ts

```diff
@@ -63,9 +64,15 @@ export class GithubDiscoveryProcessor implements CatalogProcessor {
         `There is no GitHub integration that matches ${location.target}. Please add a configuration entry for it under integrations.github`,
       );
     }
+
+    // Building the org url here so that the github creds provider doesn't need to know
+    // about how to handle the wild card which is special for this processor.
+    const { source, organization } = parseGitUrl(location.target);
```

Comment:
`organization` seems to be a CloudForge thing according to docs. Afaik best way to get owner is `owner ?? name`, although in this case you got the `parseUrl` happening just below so perhaps move that up instead?